.github/actions/security-scan/action.yaml

```diff
@@ -16,5 +27,20 @@ runs:
         format: 'table' 
         severity: 'CRITICAL'
         exit-code: '1'
+        output: 'trivy-results.sarif'
```

Comment:
We can do the same thing with the go vuln check results. (a little above this)

Reply:
I was thinking this but I was wondering what uploading two will result in on GH Security Tab... I guess as an artifact nothing?

Reply:
No clue tbh